src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -559,6 +559,14 @@ In this message, "root element" refers to the root element of an XML file.
     <value>Error parsing XML: {0}</value>
     <comment>{0} - The exception message and stack trace of the associated exception</comment>
   </data>
+  <data name="XA4231" xml:space="preserve">
+    <value>The Android class parser value '{0}' is deprecated and will be removed in a future version of Xamarin.Android. Update Project Properties to use 'class-parse'.</value>
+    <comment>The value 'class-parse' should not be translated.</comment>
+  </data>
+  <data name="XA4232" xml:space="preserve">
+    <value>The Android code generator value '{0}' is deprecated and will be removed in a future version of Xamarin.Android. Update Project Properties to use 'XAJavaInterop1'.</value>
```

Comment:
Hmm. Since "target value" is a common phrase that isn't applicable here, maybe it can be `code generation target {0}`:

```
<value>The Android code generation target '{0}' is deprecated and will be removed in a future version of Xamarin.Android. Update the project properties to use 'XAJavaInterop1'.</value>
<comment>The following are literal names and should not be translated: XAJavaInterop1
{0} - The name of the current code generation target</comment>
```